scripts/deploy-livenet.js

```diff
@@ -0,0 +1,25 @@
+const assert = require("assert");
```

Comment:
Is there any reason we cannot merge this and the `toucan-bridge.js` script?

Reply:
holy moly, i completely missed that! the name was so misleading that i didn't even had a look inside 🙈  @dave are  you okay with merging the two?

Reply:
Btw we could even merge with deploy-local and just have an extra flag `--local=true` that sets up the local environment before deploying the bridge and could also enforce that the chain id is a local one. The benefit of this being that our deploy code is tested at all times via CI.

Reply:
sounds good. i'll amend this PR 👍

Reply:
Hey sorry I totally missed this notification last week... Yes I'm fine with rearranging the JS, but ping me when the PR is ready and I'll test it with the docker testing infrastructure we have in https://github.com/rnd-inc/toucan-bridge-service ... essentially as long as deploy-local.js can be coaxed into doing what it does now, I'm happy!

Reply:
BTW deploy-local.js now no longer calls `prepareToucanEnv`, it just calls `deployFixedContracts`... so it doesn't create the initial data, it just deploys the contracts. (The tests create their own initial data now).